app/models/server/raw/LivechatRooms.js

```diff
@@ -84,15 +84,15 @@ export class LivechatRoomsRaw extends BaseRaw {
 			firstParams.push(matchUsers);
 		}
 		const params = [...firstParams, usersGroup, project];
+		if (options.sort) {
```

Comment:
Why are you checking the `options.sort` if you're not using it?
I just see the default sort, the `options.sort` is not being considered.